Dockerfile

```
@@ -1,9 +1,19 @@
 # Building the application
 FROM golang:1.16-alpine
 
-RUN apk add curl yarn && sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d -b /bin
+# Installing dependencies
+RUN apk add curl git yarn && sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d -b /bin
 WORKDIR /go/src/tldrprogress
 
+# Caching go packages
+COPY go.mod go.sum ./
+RUN go mod download
+
+# Caching yarn packages
+COPY resources/package.json resources/yarn.lock resources/
+RUN (cd resources ; yarn)
```

This is more common:
```suggestion
RUN cd resources && yarn
```